docs/source/release-notes/2021-1-1-Eos.md

```diff
@@ -0,0 +1,147 @@
+* API: `Controller.triggerHapticPulse` now supports selecting an arbitrary device instead of triggering two hands every time. (#1013)
+* Application: Non-uniformly scaled sphere zones and models now behave like ellipsoids. (#1033)
+* API: Fix `keepAspectRatio` property of Image Entities to correctly scale with any ratio. (#1050, #1141)
+* App: 'Create' app's tooltips have been fixed. (#1063)
+* Application: Fix a crash on shutdown for OSX. (#1064)
+* Application: Fix a crash on some avatars in VR due to an inverse kinematics error. (#1066)
+* App: 'Create' app has had a Broken URL Report utility added to it to aid in finding broken URLs on entity properties in a domain. (#1069, #1144)
+* Graphics: Material improvements package. (#1076) 
+    * Fix transparency issue on shape entities.
+    * Gizmo, Image, and Text entities now support material entities on them.
+    * Fix `cullFaceMode` not rendering correctly on unparented material entities.
+* Application: Rename 'Vircadia Sandbox' to 'Vircadia Server'; disable server shortcuts being created with an express install on Windows. (#1081)
+* App/API: Improved ban functionality. (#1090, #1155) 
+    * App: 'People' app now has a second ban button that kicks based on username, machine fingerprint, and IP address; the original ban button now kicks based on username and machine fingerprint.
+    * API: `Users.kick` function now offers the ability to specify a `banFlags` property to indicate what should happen when kicking a user.
+* App: 'Chat' now has domain chat muted by default. (#1094)
+* Graphics: Fix glTF vertex colors. (#1123)
+* Graphics: Fix parts of avatars rotating incorrectly. (#1124)
+
+#### Miscellaneous
+
+* Docs: Update build instructions for the Oculus Quest platform. (#800)
+* Docs: Add full packaging instructions for Linux-based servers. (#899)
+* Docs: Document addition of the `RELEASE_NAME` environment variable for versions. (#1014)
+* Application: Add names to some threads to aid in debugging. (#1040)
+* Docs: Remove nvtt workaround from the Linux build documentation.
+* Application: Optimize and tidy up some code in the `MyAvatar` class. (#1047)
+* Build: Update Linux-based server packaging scripts to allow input of versions. (#1055)
+* Build: Add the release name to the end of installers' names. (#1057)
+* Application: Set resource total size to content size when using a `qrc://` resource. (#1068)
+* Application: Fix 'General' settings privacy option text to correctly reference 'Vircadia'. (#1071)
+* Docs: Define code blocks to fix Sphinx warnings in Vircadia docs. (#1078, #1083)
+* Build: Add the ability to preload an entity script whitelist when building. (#1081)
+* Application: Implement machine fingerprints for Linux. (#1092)
+* Misc: Add 'Stale' bot configuration to the repository. (#1117)
+* Application: Fix `VIRCADIA_MEMORY_DEBUGGING` on Linux/gcc. (#1127)
+* Application: Fix a stack-use-after-return error in the `RenderableModelEntityItem` class.
```

Comment:
PR?

Reply:
Probably this https://github.com/vircadia/vircadia/pull/1132